dxapp_compliance/dxapp_queries.py

```
@@ -906,17 +907,168 @@ def get_latest_commit_date(self, organisation_name, repo_name, token):
+                    file_info = "No requirements.txt file found."
+
+            else:
+                file_exists = False
+                file_info = f"Error with status code {status_code}: {list_result.stderr}"
+
+        except subprocess.CalledProcessError as e:
+            status_code = e.returncode
+            file_exists = False
+            file_info = f"Error checking file: {e.stderr}"
+
+        except json.JSONDecodeError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)